eXo/Sources/Controllers/Basic/eXoWebBaseViewController.swift

```diff
@@ -37,6 +38,8 @@ class eXoWebBaseController: UIViewController {
     */
     func setupWebView (_ webViewContainer : UIView) {
         let wkWebViewConfiguration = WKWebViewConfiguration()
+        // Add configuration to wkwebview relevant to user agent
+        wkWebViewConfiguration.applicationNameForUserAgent = "Version/8.0.2 Safari/600.2.5"
```

Comment:
Could we add the real versions of Safari used by the device here ? without putting other hardcoded versions ?

Reply:
Here there is no problem to put a specific version of safari. it was a problem of request using wkwebview , we just force the request to use Safari with stable version